PF/Assignment3/BinaryTest.java

```diff
@@ -0,0 +1,41 @@
+package assignment3;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BinaryTest {
+	BinarySearch obj = new BinarySearch();
+	int[] arr = { 2, 5, 8, 9, 10, 77, 55 };
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void test() {
```

Comment:
ACTION: Test Method Name is not relevant. You have to create 2 diffrent test methods for when number found and when number not found.